lib/sixpack.rb

```diff
@@ -81,12 +88,9 @@ def participate(experiment_name, alternatives, force=nil, kpi=nil, traffic_fract
         params[:record_force] = record_force
       end
 
-      res = self.get_response("/participate", params)
-      # On server failure use control
-      if res["status"] == "failed"
-        res["alternative"] = {"name" => alternatives[0]}
-      end
-      res
+      response = self.get_response("/participate", params)
+      raise SixpackRequestFailed.new(response) if response["status"] == "failed"
```

Comment:
What do you think about moving this `raise` to `#get_response`?

Reply:
makes sense, I'll do it

Reply:
thanks, merge when done @drnluz